### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jkimblad**
Skeleton for a work-in-progress readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Repository for the open-source pinball game red-roses
```
Comment (pjheden) [inline README.md]: It's not a pinball game. Actually this one is heavily inspired by the "warlock"-mod from WC3

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Repository for the open-source pinball game red-roses
+
+[![Software License][ico-license]](LICENSE.md)
```
Comment (pjheden) [inline README.md:3]: Lets go with the MIT license, we are good guys!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Repository for the open-source pinball game red-roses
+
+[![Software License][ico-license]](LICENSE.md)
+
+
+## Structure
+
+The project is structured as following.
```
Comment (pjheden) [inline README.md]: as following:

File: README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Testing
+
+## Contributing
+
+Please see [CONTRIBUTING](CONTRIBUTING.md) and [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md) for details.
+
+## Security
+
+If you discover any security related issues, please email pjheden@gmail.com instead of using the issue tracker.
```
Comment (pjheden) [inline README.md]: julianheden@gmail.com

File: README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## Credits
+
+- [Julian Hedén](http://julianheden.se/)
```
Comment (pjheden) [inline README.md:37]: Nice!

**Comment by jkimblad** [comment]
Any status update?

**Review by pjheden — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jkimblad) [reply README.md:3]: Oh, sorry. I just opened an issue about this. Will close

**Review by pjheden — APPROVED** [review]

**Pull request merged**